src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
+@import '/src/styles/variables';
+
+html {
+  font-family: Roboto, serif;
+}
+
 body {
-  margin: 0;
+  margin: $margin;
```

Comment:
The variable `$margin` is used here, but it's not defined within this file. Ensure that it is defined in the imported `variables` file or elsewhere in your project.